static/app/components/assigneeSelector.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled';
+
+import {AssigneeBadge} from 'sentry/components/assigneeBadge';
+import AssigneeSelectorDropdown, {
+  type AssignableEntity,
+} from 'sentry/components/assigneeSelectorDropdown';
+import {Button} from 'sentry/components/button';
+import {t} from 'sentry/locale';
+import type {Group} from 'sentry/types/group';
+import type {User} from 'sentry/types/user';
+
+interface AssigneeSelectorProps {
+  assigneeLoading: boolean;
+  group: Group;
+  handleAssigneeChange: (assignedActor: AssignableEntity | null) => void;
+  memberList?: User[];
+}
+
+export function AssigneeSelector({
```

Comment:
What is the difference between this and `AssigneeSelectorDropdown` (which this uses?). Might be good to explain that since this is a global component.